plugins/modules/openssh_cert.py

```diff
@@ -34,13 +34,34 @@
     force:
         description:
             - Should the certificate be regenerated even if it already exists and is valid.
+            - Equivalent to I(regenerate=always).
         type: bool
         default: false
     path:
         description:
             - Path of the file containing the certificate.
         type: path
         required: true
+    regenerate:
+        description:
+            - When C(never) the task will fail if a certificate already exists at I(path) and is unreadable
+              otherwise a new certificate will only be generated if there is no existing certificate.
```

Comment:
```suggestion
            - When C(never) the task will fail if a certificate already exists at I(path) and is unreadable.
              Otherwise a new certificate will only be generated if there is no existing certificate.
```